### ISSUE TITLE
All skins are missing model / texture for Bedrock players.

### ISSUE THREAD

**Issue by sleepymincy** [GeyserMC/Geyser#4234]
### The issue:
**All skins** are purple mess for Bedrock players. **Even Bedrock skins** are like that for Bedrock players. On Java everything works as intended, I can see both Bedrock player skins, but not the other way around.

- How it looks on Bedrock (middle skin was made in Bedrock skin creator):
![Bedrock](https://github.com/GeyserMC/Geyser/assets/146667481/d22f3524-15bf-4f71-94ab-53c2abfdfc04)

- How it looks on Java:
![Java](https://github.com/GeyserMC/Geyser/assets/146667481/f4296b1a-570e-4794-acde-1108980510ac)

### A few things to know:

- I use [playit.gg](https://playit.gg/) to host a game for both Java and Bedrock versions of the game.
- I have checked Microsoft privacy settings for Bedrock players (**custom skins are allowed to be shown**).
- Not only custom skins are bugged, the skins you make with Bedrock skin creator are missing texture, too.
- I use [Floodgate](https://github.com/GeyserMC/Floodgate) and [GeyserSkinManager](https://github.com/Camotoy/GeyserSkinManager), the presence or absence of GeyserSkinManager has no effect on this issue.
- I'm using WSL with Ubuntu to host a server.
- I use [ViaVersion](https://github.com/ViaVersion/ViaVersion) since I host on 1.20.1.

### To Reproduce

1. Host server locally on WSL.
2. Use playit.gg tunnel.
3. Connect to the server.
4. Voilà!

### Expected behaviour

I expected all skins to show correctly.

### Version:

This server is running Paper version git-Paper-196 (MC: 1.20.1) (Implementing API version 1.20.1-R0.1-SNAPSHOT) (Git: 773dd72)
You are running the latest version
Previous version: git-Paper-17 (MC: 1.20)

### Plugins:
- Floodgate
- Geyser-Spigot
- GeyserSkinManager
- ViaVersion
- LeashPlayers

### Geyser Dump

https://dump.geysermc.org/jc7bedlRkMQV9do9Kubo964BebuuqkxO

### Geyser Version

Geyser version 2.2.0-SNAPSHOT (git-master-7d489c7) (Java: 1.20.2, Bedrock: 1.20.0/1.20.1 - 1.20.30/1.20.31)

### Minecraft: Bedrock Edition Device/Version

1.20.31, Windows 10, laptop.

**Comment by onebeastchris** [comment]
This usually means that Geyser/floodgate aren't able to reach the Mojang  skin api/Geyser global api for Java & Bedrock skins respectively. Do you get any errors in the logs?

**Comment by sleepymincy** [comment]
> This usually means that Geyser/floodgate aren't able to reach the Mojang skin api/Geyser global api for Java & Bedrock skins respectively. Do you get any errors in the logs?

Nope. No errors, besides some custom resourcepack warnings. I tried forcing `force-show-skins: true` in GeyserSkinManager, and it's the only error in the log that is related to anything with skins. And it's not even an error, just a warning.

**Comment by onebeastchris** [comment]
Please enable debug-mode within Geyser's config, and still send a log via mclo.gs. Additionally, with floodgate installed, what's the output of the `/floodgate firewall` command?

**Comment by sleepymincy** [comment]
- Logs with debug mode: https://mclo.gs/WKE2X3a
  - Error from [this](https://github.com/LunarEclipseStudios/From-The-Fog/) datapack and it's intended.
- `/floodgate firewall` output:

![image](https://github.com/GeyserMC/Geyser/assets/146667481/197d598d-0476-47bb-b70c-a6f5c0afdcde)

P.S. Updated Geyser to the last version after I saw warnings in the console. Problem persists.

**Comment by SEKITEI810** [comment]
I have just the same problem.

**Comment by More678** [comment]
This problem still exists now

**Comment by ElMand-o** [comment]
Same here

**Comment by onebeastchris** [comment]
Closing due to missing reproduction steps - if this still occurs, please create a new issue with them!
